kibana/connecting_client.go

```diff
@@ -84,19 +85,24 @@ func NewConnectingClient(cfg *kibana.ClientConfig) Client {
 
 // Send tries to send a request to Kibana via established connection and returns unparsed response
 // If no connection is established an error is returned
-func (c *ConnectingClient) Send(method, extraPath string, params url.Values,
+func (c *ConnectingClient) Send(ctx context.Context, method, extraPath string, params url.Values,
 	headers http.Header, body io.Reader) (*http.Response, error) {
+	span, _ := apm.StartSpan(ctx, "Send", "custom")
+	defer span.End()
 	c.m.RLock()
 	defer c.m.RUnlock()
 	if c.client == nil {
 		return nil, errNotConnected
 	}
+
 	return c.client.Send(method, extraPath, params, headers, body)
 }
 
 // GetVersion returns Kibana version or an error
 // If no connection is established an error is returned
-func (c *ConnectingClient) GetVersion() (common.Version, error) {
+func (c *ConnectingClient) GetVersion(ctx context.Context) (common.Version, error) {
+	span, _ := apm.StartSpan(ctx, "GetVersion", "custom")
```

Comment:
Can you please use "app" instead of "custom"? That way it'll get bundled up with transaction timings in the breakdown metrics. Same in SupportsVersion.